sql-checker/src/main/scala/de/thm/ii/fbs/SQLTask.scala

```diff
@@ -38,31 +37,18 @@ class SQLTask(val filepath: String, val taskId: String){
   /**
     * Class instance JDBCDriver
     */
-  final var JDBCDriver: String = config.getString("sql.driver")
+  final var JDBCDriver: String = "com.mysql.jdbc.Driver"
   /**
     * Class instance connection
     */
-  private val URL: String = config.getString("sql.connectionUri")
-  private val user: String = config.getString("sql.user")
-  private val password: String = config.getString("sql.password")
+  private val URL: String = "jdbc:mysql://mysql2:3306"
+  private val user: String = "root"
+  private val password: String = "SqfyBWhiFGr7FK60cVR2rel"
 
   private var connection: Connection = DriverManager.getConnection(URL, user, password)
   private val s = connection.createStatement()
   private val timeoutsec = 10
 
-  private def loadFactoryConfigPath() = {
-    val dev_config_file_path = System.getenv("CONFDIR") + "/../docker-config/sqlchecker/application_dev.conf"
-    val prod_config_file_path = "/usr/local/appconfig/application.config"
-
-    var config_factory_path = ""
-    if (Files.exists(Paths.get(prod_config_file_path))) {
-      config_factory_path = prod_config_file_path
-    } else {
-      config_factory_path = dev_config_file_path
-    }
-    config_factory_path
```

Comment:
I suggest an alternative approach: we change the loadFactoryConfigPath method something like this

```scala
private def loadConfig() = {
  val prodConfigFilePath = "/usr/local/appconfig/application.config"
  if (Files.exists(prodConfigFilePath)) {
    val configFile = ConfigFactory.parseFile(new File(prodConfigFilePath))
    ConfigFactory.load(appConfig)
  } else {
    ConfigFactory.load()
  }
}
```